resources/views/permissions.blade.php

```diff
@@ -0,0 +1,33 @@
+
+@extends ('layouts.main')
+	@section('content')
+
+	<table class="table">
+		<thead>
+			<th>User</th>
+			<th>Mail</th>
+			<th>Permission</th>
+			<th>Save changes</th>
+		</thead>
+		@foreach($users as $user)
+			<tbody>
+				<td>{{$user->name}}</td>
+				<td>{{$user->email}}</td>
+				{{ Form::open(array('url' => 'permission/' . $user->id)) }}
+				<td>
+					<select name ="permission">
+						<option {{ $user->permission->id == 1? "selected" :""}} value="1">Standard</option>
```

Comment:
Should not be hardcoded. You may pass the user types available from the controllers